scripts/dev.sh

```
@@ -74,7 +74,7 @@ if [ ! -d "$PACKAGES_DIR" ]; then
 fi
 
 # List of working folders to watch (relative to $PACKAGES_DIR)
-WORKING_FOLDERS=("client-direct") # Core is handled separately
+WORKING_FOLDERS=("client-direct", "plugin-initia") # Core is handled separately
```

Shellcheck (0.10.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)